website/static/css/pages/wiki-page.css

```diff
@@ -6,6 +6,9 @@
 .panel-heading.no-background {
     background: transparent;
 }
+.panel-height {
+    height: 50px;
```

Comment:
Class names should describe the UI they are applied to; they should not describe the actual CSS rules that they implement. How about `wiki-panel-header`?